css/style.css

```diff
@@ -0,0 +1,104 @@
+*,
+*::after,
+::before {
+  -webkit-box-sizing: border-box;
+  -moz-box-sizing: border-box;
+  box-sizing: border-box;
+}
+
+body {
+  font-family:
+    -apple-system,
+    BlinkMacSystemFont,
+    "Segoe UI",
+    "Roboto",
+    "Helvetica Neue",
+    Arial,
+    sans-serif;
+  font-size: 1.6rem;
+  line-height: 1.5;
+  text-align: center;
+  color: #333;
+  margin: 0;
+}
+
+#main {
+  margin: 30px 8px;
+  padding: 15px;
+  border-radius: 5px;
+  background: #eee;
+}
+
+#title h2 {
+  font-size: 2.5rem;
+}
+
+#title p {
+  font-size: 0.9rem;
+}
+
+/* ==== Image section start ==== */
+
+#img-div {
+  background: white;
+  padding: 10px;
+  margin: 0;
+  display: flex;
+  flex-direction: column;
+}
+
+#image {
+  max-width: 100%;
+  display: block;
+  height: auto;
+  margin: 0 auto;
+}
+
+#img-caption {
+  margin: 15px 0 5px 0;
+  font-size: 1rem;
+}
+
+/* ==== Image section end ==== */
+
+/* ==== Tribute Infor Start ==== */
+
+#tribute-info {
+  display: block;
+}
+
+#headline {
+  font-size: 1.1rem;
+  margin: 50px 0;
+}
+
+ul {
+  max-width: 550px;
+  margin: 0 auto 50px auto;
+  text-align: left;
+  line-height: 1.6;
+}
```

Comment:
Awesome job you did here 💯 , Please Use responsive values in your CSS rules, like percentages (i.e. width: 80%), instead of pixels, so where you had max-width: 500px, use width: with percentage value, it could be 80% or more, or less.